.circleci/config.yml

```diff
@@ -138,105 +157,110 @@ jobs:
-      jobs:
+    jobs:
       - check-code-formatting
       - build-and-test
       - assume-role-development:
           context: api-assume-role-development-context
           requires:
-              - build-and-test
+            - build-and-test
           filters:
-             branches:
-               only: development
+            branches:
+              only: development
       - terraform-init-and-apply-to-development:
-          context: api-assume-role-development-context
           requires:
-              - assume-role-development
+            - assume-role-development
           filters:
-             branches:
-               only: development
+            branches:
+              only: development
       - deploy-to-development:
-          context: api-assume-role-development-context
           requires:
             - build-and-test
           filters:
             branches:
               only: development
   check-and-deploy-staging-and-production:
-       jobs:
-       - build-and-test:
-           filters:
-             branches:
-               only: master
-       - terraform-init-and-apply-to-staging:
-           context: api-assume-role-staging-context
-           requires:
-             - build-and-test
-           filters:
-             branches:
-               only: master
-       - deploy-to-staging:
-           context: api-assume-role-staging-context
-           requires:
-             - build-and-test
-           filters:
-             branches:
-               only: master
-       - permit-production-terraform-release:
-           type: approval
-           requires:
-             - deploy-to-staging
-       - assume-role-production:
-           context: api-assume-role-production-context
-           requires:
-               - permit-production-terraform-release
-           filters:
-              branches:
-                only: master
-       - terraform-init-and-apply-to-production:
-           requires:
-             - assume-role-production
-           filters:
-             branches:
-               only: master
-       - permit-production-release:
-           type: approval
-           requires:
-             - deploy-to-staging
-           filters:
-             branches:
-               only: master
-       - deploy-to-production:
-           context: api-assume-role-production-context
-           requires:
-             - assume-role-production
-             - permit-production-release
-           filters:
-             branches:
-               only: master
+    jobs:
+      - build-and-test:
+          filters:
+            branches:
+              only: master
+      - assume-role-staging:
+          context: api-assume-role-development-context
+          requires:
+            - build-and-test
+          filters:
+            branches:
+              only: master
+      - terraform-init-and-apply-to-staging:
+          # context: api-assume-role-staging-context # needed if creds file persisted?
```

Comment:
I can test this before these changes are merged, and/or watch the CI build after merge to examine the results of these jobs that I am assuming (based on behaviour in the development-only jobs) will now work without needing the credentials provided by the context.